src/FffGcodeWriter.h

```diff
@@ -225,11 +225,11 @@ class FffGcodeWriter : public SettingsMessenger, NoCopy
     LayerPlan& processLayer(const SliceDataStorage& storage, int layer_nr, unsigned int total_layers) const;
 
     /*!
-     * Whether the extruders need to be primed separately just before they are used.
+     * Whether any extruder needs to be primed separately just before they are used.
```

Comment:
This description should note that this functions checks whether priming should happen on the first layer.

Actually an extruder should always be primed before it starts, but that priming may or may not include a poop.